### PR TITLE
feat: add how to write unit tests

### PR DESCRIPTION
**Pull request by lightClouds917**
## What is the purpose of the change

add how to write unit tests .md
#843 
除了2篇how to write unit tests .md之外，其他的都是处理冲突导致空格格式变化，可以忽略。

## Brief changelog

XXXXX

Follow this checklist to help us incorporate your contribution quickly and easily:

- [x] Make sure there is a [GITHUB_issue](https://github.com/apache/incubator-seata-website/issues) filed for the change(usually before you start working on it).  Trivial changes such as typos do not necessitate a GITHUB issue. Your pull request should solely focus on addressing this specific issue without incorporating any other modifications. Each PR should resolve a single issue.
- [ ] Format the pull request title like `doc: add an operations guide document`. Each commit within the pull request should contain a meaningful subject line and body.
- [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
- [ ] Test your code locally by running `npm run build`, and make sure it works as expected.
- [ ] Ensure that no files under the `build` folder are included in the commit, as well as the `package-lock.json` and `yarn.lock` files.

### CODE REVIEW

**Comment by lightClouds917** [comment]
中文review check没问题后，再翻译英文。

File: i18n/zh-cn/docusaurus-plugin-content-blog/how-to-write-unit-tests.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+date: 2024-02-20
+---
+
+# 背景
+随着 Seata 项目的不断发展和壮大，我们的贡献者群体也在持续扩大。项目的功能不断增强，对于代码质量的要求也在提高。在这个过程中，我们期望每一位贡献者在提交功能代码的同时，能够附带规范、完备的测试用例。  
+
+一个优秀的项目，其完备的单元测试是基本保障。Test-Driven Development（TDD）理念已经提出多年，它强调在编写功能代码之前先编写测试用例。通过编写单元测试，开发者可以更深入地理解代码中相关类和方法的作用，掌握核心逻辑，熟悉各种场景的运行情况。同时，单元测试也为开源项目提供了稳定安全的保障，使得项目在接受贡献者代码时，能够确保代码的质量和稳定性。  单元测试是质量保障的第一环，有效的单元测试能够提前发现90%以上的代码Bug问题，同时也能防止代码的腐化。在项目重构和演进过程中，单元测试起到了至关重要的作用，它能够确保重构后的代码仍然能够正常工作，不会引入新的Bug。
+
+ 在社区看来，贡献合理的测试用例代码和贡献功能代码同样重要，为了帮助开发者编写出高质量的测试用例，本文给出一些基础的规范和建议。
+# 推荐的框架
```
Comment (liuqiufeng) [inline i18n/zh-cn/docusaurus-plugin-content-blog/how-to-write-unit-tests.md]: 规范的说，应仅能有一个一级标题。
Normally, there should be only one primary title.

**Review by liuqiufeng — APPROVED** [review]
LGTM

File: i18n/zh-cn/docusaurus-plugin-content-blog/how-to-write-unit-tests.md (hunk 1 above)
Comment (lightClouds917) [reply i18n/zh-cn/docusaurus-plugin-content-blog/how-to-write-unit-tests.md]: done

**Pull request merged**